pkgs/applications/office/paperless/default.nix

```diff
@@ -0,0 +1,222 @@
+{ stdenv
+, lib
+, writeScript
+, fetchFromGitHub
+, makeWrapper
+
+, python36
+, file
+, imagemagickBig # includes ghostscript
+, optipng
+, poppler
+, tesseract
+, unpaper
+}:
+
+## Usage
+
+# ${pkgs.paperless}/bin/paperless wraps manage.py
+
+# ${pkgs.paperless}/share/paperless/setup-env.sh can be sourced from a
+# shell script to setup a Paperless environment
+
+# paperless.withConfig is a convenience function to setup a
+# configured Paperless instance. Example:
+#
+# nix-build --out-link ./paperless -E '
+# (import <nixpkgs> {}).paperless.withConfig {
+#   dataDir = /tmp/paperless-data;
+#   config = {
+#     PAPERLESS_DISABLE_LOGIN = "true";
+#   };
+# }'
+#
+# Setup DB
+# ./paperless migrate
+#
+# Consume documents in dataDir/consume
+# ./paperless document_consumer --oneshot
+#
+# Start web interface
+# ./paperless runserver --noreload localhost:8000
+
+## WSGI with gunicorn
+# Use a shell script like this:
+# source ${paperless}/share/paperless/setup-env.sh
+# PYTHONPATH=$paperlessSrc gunicorn paperless.wsgi
+
+with lib;
+let
+  paperless = stdenv.mkDerivation rec {
+    name = "paperless-${version}";
+    version = "2.6.0";
+
+    src = fetchFromGitHub {
+      owner = "danielquinn";
+      repo = "paperless";
+      rev = version;
+      sha256 = "160idy0vwzkb134axkiyj3yf85cgwggz0846w99fb7nckhx7v51m";
+    };
+
+    inherit python;
+
+    pythonEnv = python.withPackages (ps: with ps; [
+      coveralls
+      dateparser
+      dateutil
+      django
+      django-crispy-forms
+      django-cors-headers
+      django-filter
+      django_extensions
+      djangorestframework
+      factory_boy
+      filemagic
+      flake8
+      fuzzywuzzy
+      gunicorn
+      langdetect
+      pdftotext
+      pillow
+      pycodestyle
+      pyocrWithUserTesseract
+      pytest
+      pytest-django
+      pytest-env
+      # pytest_xdist # Disabled, broken in nixpkgs
```

Comment:
I think this is no longer the case.

Reply:
It's still broken: https://github.com/NixOS/nixpkgs/issues/45060#issuecomment-448397355
Could you reopen this issue?

Reply:
Now it's finally fixed.